### PR TITLE
graphql-cli 3.0.14 (new formula)

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/graphql-cli.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  test do
+    (testpath/".graphqlconfig.yaml").write <<~EOS
+      projects:
+        test:
+          schemaPath: schme.graphql
+    EOS
+
+    script = (testpath/"test.sh")
+    script.write <<~EOS
+      #!/usr/bin/expect -f
```
Comment (dawidd6) [inline Formula/graphql-cli.rb:27]: Could this be made to also support the `expect` binary from brew?

Comment (chenrui333) [reply Formula/graphql-cli.rb:27]: > Could this be made to also support the `expect` binary from brew?

Can you elaborate a little bit?

Comment (dawidd6) [reply Formula/graphql-cli.rb:27]: Use brewed `expect` instead of the system one, if the latter is not available.

Comment (SMillerDev) [reply Formula/graphql-cli.rb:27]:
```suggestion
      #!/usr/bin/env expect -f
```
This should do the trick right?

Comment (dawidd6) [reply Formula/graphql-cli.rb:27]: It should, yes. Thanks.

Comment (chenrui333) [reply Formula/graphql-cli.rb:27]: Got you! Updating it.

**Comment by chenrui333** [comment]
Go for the merge now.

**Pull request closed without merging**